### DISCUSSION TITLE
No se encuentra el archivo:

### DISCUSSION THREAD

**Discussion by Mardones567** [IIC2133-PUC/2024-2#64] [Tarea 0] [answered]
Hola, tengo un problema a compilar con make, estoy en Windows y me sale este error.

![image](https://github.com/user-attachments/assets/f0ff91a3-2370-40e5-8cc3-3f97e322c114)
![image](https://github.com/user-attachments/assets/e7c9acb3-1998-4229-b6e5-6e047ade411d)
He estado haciendo compilando con gcc y insertando en el codigo los input. Intente bajar el repositorio de nuevo y probar, pero me dio el mismo resultado.

**Comment by tomastrivino** [comment, marked as answer]
Hola,
por favor compila tu tarea en un entorno `WSL`,  ver https://github.com/IIC2133-PUC/2024-2/blob/master/Gu%C3%ADas/0.%20Setup.md

Saludos!
